app/dcs/code/static-site-hosting/page.md

```diff
@@ -21,29 +21,31 @@ You can use your own domain name and host your own static website on Storj
 
 1.  Download the uplink binary ([](docId:h3RyJymEIi4gf2S9wVJg8)) and upload your static site files to Storj DCS. You may also upload your files in any other manner, but you will need the Uplink CLI for the remaining steps.
 
-2.  Share the bucket or object prefix (not individual objects) that will be the root of your website/subdomain. At the root, name your home page`index.html`. The website will serve the index.html file automatically e.g.`http://www.example.test` and `http://www.example.test/index.html`will serve the same content. Anything shared with `--dns` will be _readonly_ and available _publicly_ (no secret key needed). You can optionally specify your preferred linkshare endpoint with `--base-url`
+2.  Share the bucket or object prefix (not individual objects) that will be the root of your website/subdomain. At the root, name your home page`index.html`. The website will serve the index.html file automatically e.g.`http://www.example.test` and `http://www.example.test/index.html`will serve the same content. Anything shared with `--dns` will be _readonly_ and available _publicly_ (no secret key needed).
+3. You can optionally specify your preferred linkshare endpoint with `--base-url`.
```

Comment:
I'm not sure many will even need to change this, unless they're running a custom linksharing. It might be best we just remove all references to `--base-url` because it defaults to `https://link.storjshare.io` anyway

Reply:
Agreed. I just left it because it was there.

Reply:
Fair enough, let's change that in another change then